core/test/functional/api/v2/admin/utils.js

```diff
@@ -7,5 +7,11 @@ module.exports = {
         getApiQuery(route) {
             return url.resolve(API_URL, route);
         }
+    },
+
+    doAuth() {
+        const args = Array.prototype.slice.call(arguments);
+        args.unshift(`${API_URL}session/`);
+        return testUtils.API.doAuth.apply(null, args);
```

Comment:
@allouis Could you please make these changes available in master tomorrow morning? 
So all functional API v2 tests can authenticate using sessions?

This code adaptions works, but e.g. i was not sure if all the tests now need to use `set('Origin', config.get(url))`.
Would be helpful to get an introduction too.

Reply:
@kirrg001 The basic `doAuth` method is in master now! Yeah, all the tests will need to set the origin - I can run you through an intro when you're up :relaxed: